src/bot.ts

```diff
@@ -168,12 +172,21 @@ export class DiscordBot {
       };
     }
     let msg = null;
+    const webhooks = await chan.fetchWebhooks();
+    const hook : Discord.Webhook = webhooks.filterArray((h) => h.name === "_matrix").pop();
     try {
-      if (botUser) {
+      if (!botUser) {
+        msg = await chan.send(embed.description, opts);
+      } else if (hook && !hasAttachment) { //Remove !hasAttachment when https://github.com/hydrabolt/discord.js/pull/1449 is fixed
+        if (hasAttachment) {
+          hookOpts.file = opts.file;
+          msg = await hook.send(embed.description, hookOpts);
```

Comment:
Looking at it, probably best to comment this out since the else aught to take care of this anyway